app/javascript/shared/helpers/uploadRules.js

```diff
@@ -0,0 +1,141 @@
+// Constants
+export const INBOX_TYPES = {
```

Comment:
Cant we use this from @app/javascript/dashboard/helper/inbox.js

Reply:
We're moving this rule to the utils repo, which is why I added it here.